### PR TITLE
feat: adding logging to validation

### PR DESCRIPTION
**Pull request by AAfghahi**
### SUMMARY
Logs the engine to the validation_parameters for better logging. 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx [hunk 1]
```diff
@@ -908,14 +908,14 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
         />
       );
     }
-
-    const message: Array<string> = Object.values(dbErrors);
+    const message: Array<string> | null =
```
Comment (AAfghahi) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx:911]: This fixes a database connection errror when test connection has failed.

**Review by betodealmeida — APPROVED** [review]
It's not related to your changes, but I think we really should show all the messages, not just the first one. Left some suggestions on how to do that.

**Review by betodealmeida — CHANGES_REQUESTED** [review]
Actually, can you run a test and see if the error messages get logged successfully?

I think @hughhhh has some experience checking that logs are produced correctly when testing locally.

**Comment by AAfghahi** [comment]
Screen Shot 2021-09-08 at 4 18 18 PM

@betodealmeida this is from my local logging.

**Review by betodealmeida — APPROVED** [review]

**Pull request merged**